### DISCUSSION TITLE
Feature Examples

### DISCUSSION THREAD

**Discussion by stockrimsn** [PeterStrick/ViVeTool-GUI#25] [Q&A] [answered]
How can I see Which feature look like What? Is there any list or  video?

**Comment by PeterStrick** [comment]
> How can I see Which feature look like What? Is there any list or video?

Do you mean you want to see what a feature does exactly? If so then that is a feature for the upcoming Version 1.7

**Comment by stockrimsn** [comment]
This app is really so beautifull for enable features but as I said, I can't find feature meanings. I'll wait 1.7 but is there any way to change Notification Menu? I'm using MyDockFinder and default notification center is so bad.

Thank you for your work.

**Comment by PeterStrick** [comment, marked as answer]
Only the Feature Scanner uses Notifications. I could add an option to change the Notification Area where it will pop up

**Reply by stockrimsn** [reply]
Yea I want to change Windows 11's Notification Pop-up. It's look like so old. I'm waiting an update for change it look like Action Center.
